components/sidebar.jsx

```diff
@@ -71,7 +80,7 @@ var Sidebar = React.createClass({
           <Link to="home">
             <img src="/img/nav/mozilla-wordmark-white.svg" alt="Webmaker logo" className="moz-logo"/>
           </Link>
-          <span aria-label="toggle" role="button" onKeyUp={this.handleHamburgerClick} onKeyDown={this.handleHamburgerClick} className="glyphicon glyphicon-menu-hamburger hidden-lg hidden-md"
+          <span aria-label="toggle" role="button" onKeyDown={this.handleHamburgerKeyDown} className="glyphicon glyphicon-menu-hamburger hidden-lg hidden-md"
```

Comment:
Oh, I am realizing now that the title of this PR might be a bit misleading. It's called "only toggle hamburger on enter/space" because at present, pressing *any* key toggles the hamburger. It doesn't mean "only enter/space should toggle the hamburger, ignore all other input devices" :)